GameServerApp/Program.cs

```diff
@@ -17,13 +17,17 @@ public class ArgsOptions
         [Option("config", DefaultValue = "Settings/GameInfo.json")]
         public string ConfigPath { get; set; }
 
+        [Option("configJSON", DefaultValue = "")]
+        public string ConfigJSON { get; set; }
+
         [Option("port", DefaultValue = (ushort)5119)]
         public ushort ServerPort { get; set; }
 
         public static ArgsOptions Parse(string[] args)
         {
             var options = new ArgsOptions();
             Parser.Default.ParseArguments(args, options);
+            System.Console.WriteLine("Port is: " + options.ServerPort);
```

Comment:
I don't think there should be a console print here. If you want to print the port number, please place it to the GameServerLib somewhere, and use the logger instead of direct print.